### PR TITLE
Deploying/Releasing process made simpler

### PR DESCRIPTION
**Pull request by chmeldax**
- Travis will automatically push gem to RubyGems
- Version.rb introduced
- We can use `gem-release` (as a global gem) to do the bumps of versions.

Please review @simplybusiness/migration-tech-charter 

See issue: https://github.com/simplybusiness/kitcat/issues/19

### CODE REVIEW

**Comment by chmeldax** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;otSfwgNb&#x2F;45-tech-issue-19-add-autodeploy-to-rubygems"> [Tech] Issue #19: Add autodeploy to RubyGems</a>

**Comment by ljones140** [comment]
signed-off-by @ljones140 :shipit:

File: .travis.yml [hunk 1]
```diff
@@ -2,20 +2,26 @@ sudo: false
...
 install:
-  - bundle install --retry=3
+- bundle install --retry=3
 script:
-  - bundle exec rake
+- bundle exec rake
+deploy:
+  provider: rubygems
+  api_key:
+    secure: rsAAPd+SK72E8V5kphNomXGuE7VET/fP51P7JX+TzFfiZEQn7jQiqCEOtRcAiJYIroAZR834Z0BT0DGIYp92Dfh5LvLaaYSN2KJwZU6VhAaRvU7czGFT+4ScJuqqZS1sBn1L2YNgM/dTQ2BNq1mJE22ezwcBzFuenMuLAQf6obNV3FEdn/Z2WLe0FsBvsb6B/PHrvCeZo3NSmYuiwMBbdUEdL3W2ffrxacr6ErWNfPro7DuxKwJXPiDXrLam5tv6rmkASHgp34546LkZrWH5ny8G5qBJHXQh6yCGxu9XNXp3AMQ/i8H9w0E5WDXFo9l+gTAiWBEOp4zR3Arxjz3msp9tjmhAgyITZNpeiSgd4ltKTIOFinFg+R2H2HG7j4vDGfdUqBuHp30UUeWMPybJ62YcPqvF5nBzGG26U9Gv7RTysnZPRFM1hZH+F7Dj5HNBFK7bF64vb8d5AmYzWIxkkncjDciZmNfZo7bBaa6HJ/H8UpiCvAboyNptsp8WkN9vnZAcfY2h+yrWYRkFK4HF9vlGKibwWYVT3AUgLwr3+LdrXJxIFponqf6lqKVNLOXfJ9TM3S8Bs7P/cYSabXlDTaNA/x+qoLmadL/y6fqKKQ7+ZzTdXgHkuDldglaXRH8Dkv6XHo/On2/kGJBAnnFT2G9GAo0nNovfLU3RuGfCeIg=
```
Comment (joshuafleck) [inline .travis.yml:23]: Is it safe to be storing this key in a public repo?

Comment (chmeldax) [inline .travis.yml:23]: That's an encrypted version of it (https://docs.travis-ci.com/user/encryption-keys/).

**Pull request closed without merging**